Index: rpmlint-1.2/TagsCheck.py
===================================================================
--- rpmlint-1.2.orig/TagsCheck.py
+++ rpmlint-1.2/TagsCheck.py
@@ -778,7 +778,7 @@ class TagsCheck(AbstractCheck.AbstractCh
             if not valid_license:
                 self._unexpanded_macros(pkg, 'License', rpm_license)
 
-        for tag in ('URL', 'DistURL', 'BugURL'):
+        for tag in ('URL',  'BugURL'):
             if hasattr(rpm, 'RPMTAG_%s' % tag.upper()):
                 url = pkg[getattr(rpm, 'RPMTAG_%s' % tag.upper())]
                 self._unexpanded_macros(pkg, tag, url, is_url = True)
@@ -1094,7 +1094,7 @@ once.''',
 '''This rpm requires a specific release of another package.''',
 
 'no-url-tag',
-'''The URL tag is missing.''',
+'''The URL tag is missing. Please add a http or ftp link to the project location.''',
 
 'name-repeated-in-summary',
 '''The name of the package is repeated in its summary.  This is often redundant
