From 14206363857dc8ba57af605de52b0244c4e22ad4 Mon Sep 17 00:00:00 2001
From: David Greaves <david.greaves@jolla.com>
Date: Mon, 23 Mar 2020 14:42:20 +0000
Subject: [PATCH 7/7] Add a "TreatErrorsAsWarnings" configuration option

Signed-off-by: David Greaves <david.greaves@jolla.com>
---
 rpmlint/lint.py | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/rpmlint/lint.py b/rpmlint/lint.py
index 1ec2b20..d769a19 100644
--- a/rpmlint/lint.py
+++ b/rpmlint/lint.py
@@ -93,7 +93,13 @@ class Lint(object):
                             f'{self.output.score} badness; has taken {duration:.1f} s', '=')
         print(f'{quit_color}{msg}{Color.Reset}')
 
-        return retcode
+        if not self.config.configuration['TreatErrorsAsWarnings']:
+            return retcode
+        else:
+            if retcode:
+                msg = string_center('TreatErrorsAsWarnings set in options so returning success instead of {}'.format(retcode), '=')
+                print(f'{quit_color}{msg}{Color.Reset}')
+            return 0
 
     def run(self):
         try:
-- 
2.20.1

