From 47065c6edf168fcc2ad7fc72c85ed3d99dcc1468 Mon Sep 17 00:00:00 2001
From: David Greaves <david.greaves@jolla.com>
Date: Wed, 18 Mar 2020 18:58:07 +0000
Subject: [PATCH 6/7] Say what config files are causing problems

Signed-off-by: David Greaves <david.greaves@jolla.com>
---
 rpmlint/config.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpmlint/config.py b/rpmlint/config.py
index 60266fc..c9eb984 100644
--- a/rpmlint/config.py
+++ b/rpmlint/config.py
@@ -88,7 +88,7 @@ class Config(object):
         try:
             cfg = toml.load(self.conf_files)
         except toml.decoder.TomlDecodeError as terr:
-            print_warning(f'(none): W: error parsing configuration files: {terr}')
+            print_warning(f'(none): W: error parsing configuration files: {self.conf_files} : {terr}')
             cfg = None
         self.configuration = cfg
 
-- 
2.20.1

